src/ts/transformers/har.ts

```diff
@@ -30,11 +31,10 @@ import {
  * @param  {Har} harData - raw hhar object
  * @returns WaterfallDocs
  */
-export function transformDoc(harData: Har): WaterfallDocs {
+export function transformDoc(harData: Har | Log): WaterfallDocs {
```

Comment:
Adjust typing to the the one used in `@typing/har-format` - had perviously incorrectly labeled `Log` as `Har`.